platforms/android/demo/src/main/java/com/mapzen/tangram/android/MainActivity.java

```diff
@@ -88,6 +191,42 @@ public void onLowMemory() {
         view.onLowMemory();
     }
 
+    @Override public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+        int selectedIndex = 7; // custom
```

Comment:
This smells like trouble. `onItemClick` and `onItemSelected` seem overly complicated and dependent on data that's defined elsewhere. As in my earlier comment, I'd suggest removing the `Spinner` element entirely.